src/common/_modules/application_gateway/main.tf

```
@@ -647,6 +647,11 @@ module "app_gw" {
           backend               = "platform-api-gateway",
           rewrite_rule_set_name = "rewrite-rule-set-api-app-rewrite-platform-legacy"
         },
+        api-gateway-platform-cdc = {
+          paths                 = ["/api/cdc/*"]
+          backend               = "platform-api-gateway",
+          rewrite_rule_set_name = "rewrite-rule-set-api-app-rewrite-platform-legacy"
```

We don't need path rewrite rule for this routing
```suggestion
          rewrite_rule_set_name = "rewrite-rule-set-api-app"
```